ui/StatusQ/src/StatusQ/Controls/StatusPickerButton.qml

```diff
@@ -5,71 +5,65 @@ import QtQuick.Layouts 1.15
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
 
-Button {
+StatusButton {
```

Comment:
The main purpose here is to allow having a tooltip over a disabled (non-interactive) button